api/libproject/devstart.rb

```diff
@@ -5,6 +5,8 @@
 require "tempfile"
 require "fileutils"
 
+ENV["UID"] = "#{Process.euid}"
```

Comment:
@dmohs sticking this here is kind of a hack. Would it be better in the Common lib, or somewhere else?

Reply:
I set up https://github.com/all-of-us/workbench/blob/master/libproject/workbench.rb for things that don't quite make it to Common, if you want to put it there. I wouldn't want to have the common stuff do anything too magical on import, but I think copying a value to ENV is not going to confuse us.

Reply:
Moving to `workbench.rb` -- thanks @markfickett